app/services/albums.js

```diff
@@ -0,0 +1,13 @@
+const rp = require('request-promise');
+
+module.exports.listAlbums = () => {
+  rp('https://jsonplaceholder.typicode.com/albums')
+    .then(response => console.log(response))
+    .catch(err => console.log(err.message));
```

Comment:
Use `logger.error` and throw a known error

Reply:
Done :+1: